omod/src/main/resources/config.xml

```diff
@@ -145,16 +145,16 @@
 		<property>@MODULE_ID@.sTarvAdultoInitialAEncounterTypeUuid</property>
 		<defaultValue>e278f820-1d5f-11e0-b929-000c29ad1d07</defaultValue>
 		<description>
-			UUID for the S.TARV: ADULTO INICIAL A encounter type
+			UUID for the S.TARV: ADULTO INITIAL A encounter type
 		</description>
 	</globalProperty>
 
 	<globalProperty>
-		<property>@MODULE_ID@.sTarvPediatriaInicialAEncounterTypeUuid
+		<property>@MODULE_ID@.sTarvPediatriaInitialAEncounterTypeUuid
 		</property>
 		<defaultValue>e278fa8c-1d5f-11e0-b929-000c29ad1d07</defaultValue>
 		<description>
-			UUID for the S.TARV: PEDIATRIA INICIAL A encounter type

```

Comment:
This is meant to be `INICIAL` since this is Portuguese label for the partners. Please revert

Reply:
wow, i had missed that, although language support is through localisation else we code names in Portuguese naming

Reply:
I don't think localisation applies to the labels of admin fields which these are for, which is why we coded them in portuguese.